Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -356,8 +356,8 @@ def build_model_data(name, is_public, tlp, tags, intelligence, description):
     """
         Builds data dictionary that is used in Threat Model creation/update request.
     """
-    data = {k: v for (k, v) in (('name', name), ('is_public', is_public), ('tlp', tlp), ('description', description))
-            if v}
+    data = {k: v for (k, v) in (('name', name), ('is_public', is_public), ('tlp', tlp),
```

Comment:
@bakatzir this change will fix the description for `Tipeport` but will break description for all other ThreatModel like Actors, Campaign etc. Need to check if the threat model is `Tipeport` and only then replace it with `body` parameter as all the other model receive description.

Reply:
@IkaDemisto correct. fixed it, tried for vulnerability, ttp and tipreport and looks good.

Reply:
@bakatzir perfect